Tests/Unit/EventStore/Normalizer/ValueObjectNormalizerTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\EventSourcing\Tests\Unit\EventStore\Normalizer;
+
+use Neos\EventSourcing\EventStore\Normalizer\ValueObjectNormalizer;
+use Neos\Flow\Tests\UnitTestCase;
+
+class ValueObjectNormalizerTest extends UnitTestCase
+{
+    /**
+     * @test
+     * @dataProvider provideSourceDataAndClassNames
+     */
+    public function ValueObjectNormalizer_supports_denormalization_of($sourceData, string $className): void
```

Comment:
I prefer the underscored naming style, but it's not in line with our CGL and the rest of the tests currently, so I'd quickly go ahead and adjust them